x-pack/platform/plugins/shared/ml/public/application/datavisualizer/datavisualizer_selector.tsx

```diff
@@ -95,123 +99,85 @@ export const DatavisualizerSelector: FC = () => {
-                />
-              }
-              description={
-                <FormattedMessage
-                  id="xpack.ml.datavisualizer.selector.importDataDescription"
-                  defaultMessage="Import data from a log file. You can upload files up to {maxFileSize}."
-                  values={{ maxFileSize }}
-                />
-              }
-              footer={
-                <EuiButton
-                  target="_self"
-                  onClick={() => navigateToPath('/filedatavisualizer')}
-                  data-test-subj="mlDataVisualizerUploadFileButton"
-                >
-                  <FormattedMessage
-                    id="xpack.ml.datavisualizer.selector.uploadFileButtonLabel"
-                    defaultMessage="Select file"
-                  />
-                </EuiButton>
-              }
-              data-test-subj="mlDataVisualizerCardImportData"
-            />
-          </EuiFlexItem>
-          <EuiFlexItem>
-            <EuiCard
-              hasBorder
-              icon={<EuiIcon size="xxl" type="dataVisualizer" />}
-              title={
-                <FormattedMessage
-                  id="xpack.ml.datavisualizer.selector.selectDataViewTitle"
-                  defaultMessage="Visualize data from a data view"
-                />
-              }
-              description={''}
-              footer={
-                <EuiButton
-                  target="_self"
-                  onClick={() => navigateToPath('/datavisualizer_index_select')}
-                  data-test-subj="mlDataVisualizerSelectIndexButton"
-                >
-                  <FormattedMessage
-                    id="xpack.ml.datavisualizer.selector.selectDataViewButtonLabel"
-                    defaultMessage="Select data view"
-                  />
-                </EuiButton>
-              }
-              data-test-subj="mlDataVisualizerCardIndexData"
-            />
-          </EuiFlexItem>
+        <EuiFlexGroup direction="column">
           {isEsqlEnabled ? (
             <EuiFlexItem>
-              <EuiCard
-                hasBorder
-                icon={<EuiIcon size="xxl" type="dataVisualizer" />}
-                title={
-                  <EuiTextAlign textAlign="center">
-                    <>
-                      <FormattedMessage
-                        id="xpack.ml.datavisualizer.selector.selectESQLTitle"
-                        defaultMessage="Visualize data using ES|QL"
-                      />{' '}
-                      <EuiBetaBadge
-                        label=""
-                        iconType="beaker"
-                        size="m"
-                        color="hollow"
-                        tooltipContent={
+              <EuiPanel hasShadow={false} hasBorder>
+                <EuiFlexGroup alignItems="center">
+                  <EuiFlexItem>
+                    <EuiImage size="fullWidth" src={esqlImage} alt={'ES|QL input image'} />
```

Comment:
This might look better at a fixed width? For larger screen widths, the image is too prominent IMO.

Reply:
What do you think about setting a `max-width` for this image - I tried `800px`?